examples/eepromfstest/Makefile

```diff
@@ -1,35 +1,26 @@
-PROG_NAME = eepromfstest
+BUILD_DIR = build
+include $(N64_INST)/include/n64.mk
 
-ROOTDIR = $(N64_INST)
-GCCN64PREFIX = $(ROOTDIR)/bin/mips64-elf-
+.DEFAULT_GOAL := all
 
-CC = $(GCCN64PREFIX)gcc
-AS = $(GCCN64PREFIX)as
-LD = $(GCCN64PREFIX)ld
-OBJCOPY = $(GCCN64PREFIX)objcopy
-N64TOOL = $(ROOTDIR)/bin/n64tool
-CHKSUM64 = $(ROOTDIR)/bin/chksum64
+src = eepromfstest.c
+objs = $(src:%.c=$(BUILD_DIR)/%.o)
+deps = $(src:%.c=$(BUILD_DIR)/%.d)
 
-ASFLAGS = -mtune=vr4300 -march=vr4300
-CFLAGS = -std=gnu99 -march=vr4300 -mtune=vr4300 -O2 -Wall -Werror -I$(ROOTDIR)/mips64-elf/include
-LDFLAGS = -L$(ROOTDIR)/mips64-elf/lib -ldragon -lc -lm -ldragonsys -Tn64.ld --gc-sections
-N64TOOLFLAGS = -l 1M -h $(ROOTDIR)/mips64-elf/lib/header -t "EEPROMFS Test"
+all: eepromfstest_4k.z64 eepromfstest_16k.z64
+.PHONY: all
 
-ifeq ($(N64_BYTE_SWAP),true)
-$(PROG_NAME).v64: $(PROG_NAME).z64
-	dd conv=swab if=$^ of=$@
-endif
+eepromfstest_4k.z64: N64_ROM_TITLE = "EEPROMFS Test 4k"
+eepromfstest_4k.z64: N64_ED64ROMCONFIGFLAGS = --savetype eeprom4k
```

Comment:
I think the precedent of N64_ROM_TITLE was an attempt to hide the various step of ROM building to beginners (more about this later).

I would prefer to see a N64_SAVETYPE=eeprom4k here. You can still have a N64_ED64ROMCONFIGFLAGS behind the hood that by default uses N64_SAVETYPE, so that advanced users can change it, but for simple use cases I think just specifying the save type here would be easier